draft-ramseyer-grow-peering-api.md

```diff
@@ -402,12 +416,24 @@ BGP Session
   * local_ip (IP of requestor, v4 or v6)
   * peer_asn (server ASN)
   * peer_ip (server-side IP)
+  * local_bgp_role (BGP role according to {{?RFC9234}})
+  * peer_bgp_role (BGP role according to {{?RFC9234}})
+  * local_insert_asn (optional, to support route servers, defaults to true)
+  * peer_insert_asn (optional, to support route servers, defaults to true)
+  * local_monitoring_session (optional, to support monitoring systems, defaults to false)
+  * peer_monitoring_session (optional, to support monitoring systems, defaults to false)
   * peer_type (public or private)
-  * md5 (optional, as defined above)
+  * session_secret (optional, as defined above)
   * location (Peering Location, as defined above)
   * status (Session Status, as defined above)
   * session_id (of individual session and generated by the server)
 
+As not all elements are reflected in the {{autopeer}} OpenAPI definition to date, we define the missing fields here to be reflected in {{autopeer}} in the future.
```

Comment:
This looks good.  

As an aside, we should probably update the OpenAPI spec and move it in to the IETF draft as well (also brought up in https://github.com/bgp/draft-ietf-peering-api/pull/30), but that's outside the scope of this PR.

Reply:
> As an aside, we should probably update the OpenAPI spec and move it in to the IETF draft as well (also brought up in https://github.com/bgp/draft-ietf-peering-api/pull/30), but that's outside the scope of this PR.

I think that’s a very reasonable approach. RFCs should be self-contained. I’m happy to help with editing tasks like this.

Reply:
Thanks!  We'd welcome the help.